iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/VariationParameters.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2023. , All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.iidm.modification.scalable;
+
+public class VariationParameters {
```

Comment:
What is the difference with already existing ScalingParameter ? Why "variation" ?

Reply:
I thought the variation parameters were a bit more global but I can merge the two.
"Variation" was the name given in GridSuite

Reply:
We have to stay consistent in powsybl